manuals/1.0/en/types-utility.md

```
@@ -0,0 +1,368 @@
+layout: docs-en
+title: PHPDoc Utility Types
+category: Manual
+permalink: /manuals/1.0/en/types-utility.html
+---
+# PHPDoc Utility Types
+
+Utility types are used to manipulate existing types or dynamically generate new types. Using these types enables more flexible and expressive type definitions.
+
+## Table of Contents
+
+1. [key-of<T>](#key-oft)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)